drivers/modem/wncm14a2a.c

```diff
@@ -1456,7 +1462,7 @@ static int wncm14a2a_init(struct device *dev)
 		}
 
 		gpio_pin_configure(ictx.gpio_port_dev[i], pinconfig[i].pin,
-			   GPIO_DIR_OUT);
+			   GPIO_OUTPUT);
```

Comment:
Even though the driver specifies the raw active value, I think this should still incorporate the devicetree flags, since there may be PULL configurations that are important.

Reply:
I completely agree. Fixed now.